.github/workflows/scheduled_weekly_build.yml

```diff
@@ -0,0 +1,32 @@
+on:
+
+  schedule:
+    - cron: '0 0 * * 2'  # Midnight on every Tuesday
+
+name: Scheduled weekly build
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Get current date
+        run: echo "::set-output name=todays_date::$(date +'%Y-%m-%d')"
+      - name: Get latest tag
+        run: |
+          latest_tag_version_value=$(./scripts/find_latest_tag_version.sh)
+          echo "::set-output name=latest_tag_version::${latest_tag_version_value}"
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: superset-${{outputs.latest_tag_version.value}}-weekly-build-${{ outputs.todays_date.value }}
+          release_name: superset-${{outputs.latest_tag_version.value}}-weekly-build-${{ outputs.todays_date.value }}
+          body: |
+            ##todo: list of commits
```

Comment:
lgtm. Do we need to fill this part in before accepting?

Reply:
Personally, I try to limit the use of actions because they are not portable to another CI and very often they only add extra complexity to the scripts. If you want to understand now how this workflowsI works, you need to know bash and advanced features of Github Action. 

Instead, you can use github CLI, which will allow these scripts to run locally as well, if you have the required permissions. For example, see:
https://github.com/apache/airflow-site/blob/1b26f4fcd3a2eb5aa22effa41aaa65a075583d92/.github/workflows/build.yml#L99-L119